server/README.md

```diff
@@ -8,7 +8,9 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-In order to install this app it is necessary to have - Ruby Version 2.6.5 and Rails 5.2
+
+In order to install this app it is necessary to have - Ruby Version 2.6.5 and Rails 6.0.2.2
+
```

Comment:
```suggestion
In order to install this app it is necessary to have - Ruby Version 2.6.5 and Rails 6.0.2.2
```

This change I'm marking here is a GitHub Suggestion. You can turn that into a commit using the Web interface.

Try that out!

Before you accept my edit, you can name the commit by changing its commit message.

Reply:
i have wrote the commit message